src/prefect/cli/kubernetes.py

```diff
@@ -24,9 +24,14 @@
 )
 app.add_typer(kubernetes_app)
 
+# Deprecate kubernetes maniest
 manifest_app = PrefectTyper(
     name="manifest",
-    help="Commands for generating Kubernetes manifests.",
+    help="Deprecated. Use the 'Helm charts' instead.",
+    deprecated=True,
+    deprecated_name="prefect kubernetes manifest",
+    deprecated_start_date="4th May 2023",
+    deprecated_help="Use the 'Helm charts' instead.",
```

Comment:
I think we can give a better suggestion here — maybe a link to the Helm charts?

cc @jawnsy